content/_data/changelogs/lts.yml

```diff
@@ -1811,6 +1811,196 @@
+    message: >
+      Redesigned login, signup, and <em>Jenkins is (re)starting</em> pages.
+      Existing page decorators like Simple Theme Plugin will no longer work with these redesigned pages.
+    references:
+    - issue: 50447
+    - title: announcement blog post
+      url: https://jenkins.io/blog/2018/06/27/new-login-page/
+  - type: major rfe
+    pull: 3271
+    references:
+    - issue: 32442
+    - issue: 32776
+    - url: https://jenkins.io/blog/2018/07/02/new-api-token-system/
+      title: blog post
+    message: >
+      Replace single per-user API token with new system of multiple, revocable, unrecoverable API tokens with usage tracking.
+  - type: major rfe
+    pull: 3188
+    issue: 48480
+    message: >
+      The deprecated Jenkins CLI Protocol versions 1 and 2, and Java Web Start Agent Protocol versions 1, 2, and 3 have been disabled.
+      If you still use these protocols (e.g. remoting-based CLI, or old <code>slave.jar</code>s on agents), you need to re-enable these protocols after upgrade, or upgrade the clients.
+      The same recommendations as in <a href="https://jenkins.io/doc/upgrade-guide/2.121/#remoting-update">The 2.121.x upgrade guide for remoting changes</a> apply here.
+  - type: major rfe
+    pull: 3356
+    message: >
+      Check SHA-512 or SHA-256 checksums of update site and tool installer metadata and core and plugin downloads if the update site provides them.
+  - type: major rfe
+    issue: 50336
+    pull: 3370
+    message: >
+      Optional extensions are now loaded without requiring to restart Jenkins after installing an optional dependency.
+
+  - type: rfe
+    references:
+    - pull: 3422
+    - pull: 3497
+    - title: full changelog
+      url: https://github.com/jenkinsci/winstone/blob/master/CHANGELOG.md#4
+    - title: Jetty 9.4.11 changelog
+      url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.11.v20180605
+    - title: Jetty 9.4.10 changelog
+      url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.10.v20180503
+    - title: Jetty 9.4.9 changelog
+      url: https://github.com/eclipse/jetty.project/releases/tag/jetty-9.4.9.v20180320
+    - title: JMX Documentation for Jetty
+      url: https://www.eclipse.org/jetty/documentation/9.4.x/jmx-chapter.html
+    - title: full list of options
+      url: https://github.com/jenkinsci/winstone#command-line-options
+    message: >
+      Upgrade Winstone from 4.2 to 4.4 to update Jetty from 9.4.8.v20171121 to 9.4.11.v20180605, adding an option to enable JMX when running Jenkins using <code>java -jar jenkins.war</code>.
+
+  - type: rfe
+    references:
+    - issue: 51818
+    - issue: 52204
+    - url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#322
+      title: Remoting 3.22 changelog
+    pull: 3514 # and 3530
+    message: >
+      Upgrade Remoting from 3.21 to 3.23 to have agents check availability of the master's TCP Agent Listener port when connecting over TCP.
```

Comment:
This message may be confusing. Remoting 3.25 is the bundled version, `3.21.1` was used in the previous LTS (with Security backports)

Reply:
Thanks. I hate these dependency updates between LTS, and doesn't help that updates from security fixes don't get separate changelog entries (and I have only myself to blame ☹️ )